### DISCUSSION TITLE
Can't start any osu!lazer version after 2021.623.0

### DISCUSSION THREAD

**Discussion by rust2** [ppy/osu#15103] [Q&A] [answered]
So like the title says any of newer version (newer than 2021.623.0) are not starting on my pc
Most bad thing that it every time downloads latest version in background and trying to start it on next lazer startup
[https://pastebin.com/7WSXhrvq](runtime.log)

**Comment by gagahpangeran** [comment, marked as answer]
From logs

```
Microsoft Windows NT 6.1.7601 Service Pack 1
```

```
System.DllNotFoundException: Unable to load DLL 'realm-wrappers' or one of its dependencies: Не найдена указанная процедура.
```

Based on [this table](https://en.wikipedia.org/wiki/Windows_NT#Releases) and also the error above, I guess your OS is windows 7, can you confirm?

If so, we already drop support for windows 7.

**Reply by rust2** [reply]
Sure. sad
